hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/PairedStreamBuilder.java

```diff
@@ -0,0 +1,597 @@
+import com.hedera.node.app.service.token.records.TokenCreateStreamBuilder;
+import com.hedera.node.app.service.token.records.TokenMintStreamBuilder;
+import com.hedera.node.app.service.token.records.TokenUpdateStreamBuilder;
+import com.hedera.node.app.service.util.impl.records.PrngStreamBuilder;
+import com.hedera.node.app.spi.workflows.HandleContext;
+import com.hedera.node.app.spi.workflows.record.ExternalizedRecordCustomizer;
+import com.hedera.node.app.spi.workflows.record.StreamBuilder;
+import com.hedera.node.app.workflows.handle.record.RecordStreamBuilder;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.time.Instant;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * A temporary implementation of {@link StreamBuilder} that forwards all mutating calls to an
+ * {@link BlockStreamBuilder} and a {@link RecordStreamBuilder}.
+ */
+public class PairedStreamBuilder
+        implements StreamBuilder,
+                ConsensusCreateTopicStreamBuilder,
+                ConsensusSubmitMessageStreamBuilder,
+                CreateFileStreamBuilder,
+                CryptoCreateStreamBuilder,
+                CryptoTransferStreamBuilder,
+                ChildStreamBuilder,
+                PrngStreamBuilder,
+                ScheduleStreamBuilder,
+                TokenMintStreamBuilder,
+                TokenBurnStreamBuilder,
+                TokenCreateStreamBuilder,
+                ContractCreateStreamBuilder,
+                ContractCallStreamBuilder,
+                ContractUpdateStreamBuilder,
+                EthereumTransactionStreamBuilder,
+                CryptoDeleteStreamBuilder,
+                TokenUpdateStreamBuilder,
+                NodeStakeUpdateStreamBuilder,
+                FeeStreamBuilder,
+                ContractDeleteStreamBuilder,
+                GenesisAccountStreamBuilder,
+                ContractOperationStreamBuilder,
+                TokenAccountWipeStreamBuilder,
+                CryptoUpdateStreamBuilder,
+                NodeCreateStreamBuilder,
+                TokenAirdropStreamBuilder {
+    private final BlockStreamBuilder blockStreamBuilder;
+    private final RecordStreamBuilder recordBuilder;
+
+    public PairedStreamBuilder(
+            @NonNull final ReversingBehavior reversingBehavior,
+            @NonNull final ExternalizedRecordCustomizer customizer,
+            @NonNull final HandleContext.TransactionCategory category) {
+        recordBuilder = new RecordStreamBuilder(reversingBehavior, customizer, category);
+        blockStreamBuilder = new BlockStreamBuilder(reversingBehavior, customizer, category);
+    }
+
+    @Override
+    public StreamBuilder stateChanges(@NonNull List<StateChange> stateChanges) {
+        blockStreamBuilder.stateChanges(stateChanges);
+        return this;
+    }
+
+    public BlockStreamBuilder ioBlockItemsBuilder() {
```

Comment:
Nit:
```suggestion
    public BlockStreamBuilder blockItemsBuilder() {
```

No need to worry about tiny changes like this unless other bigger changes are required though

Reply:
Renamed these. 👍